buildspec.yml

```diff
@@ -0,0 +1,33 @@
+version: 0.2
+
+phases:
+  build:
+    commands:
+      - git clone https://github.com/crungehottman/exodus-lambda.git
+      - cd exodus-lambda
+      - git checkout codebuild
+      - gpg --import .whitelist.gpg
+      - LAST_TAGGED=$(git rev-list --tags --max-count=1)
+      - git verify-tag $(git describe $LAST_TAGGED)
+      - git checkout $LAST_TAGGED
+      - pip install --require-hashes -r requirements.txt --target ./package
+      - pip install --no-deps --target ./package .
+      - cp ./configuration/exodus-lambda-deploy.yaml ./package
+      - >-
+        jq -n "{
+           \"table\": {
+             \"name\": \"exodus-cdn-$ENV_TYPE\", region: \"us-east-1\"
+           },
```

Comment:
This wasn't mentioned anywhere but I think region should be an environment variable also. The pipeline will need to be aware of the AWS region anyway so allowing it to fill this value like it will elsewhere can protect against inconsistencies.